packages/esm-appointments-app/src/appointments/common-components/appointments-table.scss

```diff
@@ -1,5 +1,8 @@
 @use '@carbon/styles/scss/spacing';
-@use '@carbon/styles/scss/type';
+@use '@carbon/colors';
+@use '@carbon/layout';
+@use '@carbon/type';
+@import '~@openmrs/esm-styleguide/src/vars';
```

Comment:
All this scss is existing scss that I moved around as needed as I moved some divs and containers around.  My intent was only to copy over scss that was used.

Reply:
We don't need both `@carbon/styles/scss/spacing` and `@carbon/layout`. I'd prefer keeping `@carbon/layout` and refactoring tokens using the `spacing` namespace to use `layout` instead. So `spacing.$spacing-x` changes to `layout.$spacing-x`.

Reply:
just saw this, will handle in a post-merge commit